cmd/kube-apiserver/app/server.go

```diff
@@ -213,6 +213,22 @@ func newEtcd(etcdConfigFile string, etcdServerList util.StringList, storageVersi
 	return master.NewEtcdHelper(client, storageVersion, pathPrefix)
 }
 
+// Tells the service manager like systemd about startup. For further docs:
+// http://www.freedesktop.org/software/systemd/man/sd_notify.html
+func notifyReady() {
```

Comment:
Hmmm. Maybe I'm completely confused, but it would appear that ```APIServer.Run()``` is being called directly from here:

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cmd/kube-apiserver/apiserver.go#L44